.github/workflows/ci.yml

```diff
@@ -0,0 +1,124 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - release-[6-9].*
+  pull_request:
+    branches-ignore:
+      - release-[0-5].*
+  pull_request_target:
+    branches-ignore:
+      - release-[0-5].*
+  workflow_dispatch:
+env:
+  DOTNET_NOLOGO: true
+jobs:
+  build:
+    environment:
```

Comment:
Why did you add the environment? We aren't currently using these for anything because they aren't useful enough to replace Octopus, so it would probably be a good idea to remove this.

Reply:
I removed the environment section.  It was a holdover from the forked branch.

Reply:
You say you removed it, but it's still here.